elftools/elf/structs.py

```diff
@@ -372,6 +372,17 @@ def _create_note(self, e_type=None):
                 String('pr_psargs', 80),
             )
 
+        self.Elf_Nt_File = Struct('Elf_Nt_File',
```

Comment:
Can you please add a comment here explaining where this structure came from (link to some documentation)

Reply:
Sorry for the delay; been very busy recently. I have added a link